tools/run_tests/run_interop_tests.py

```diff
@@ -56,7 +56,7 @@
 _CLOUD_TO_CLOUD_BASE_ARGS = [
     '--server_host_override=foo.test.google.fr']
 
-# TOOD(jtattermusch) wrapped languages use this variable for location
+# TOOD(jtattermusch) wrapped languages use this variable for location 
```

Comment:
Alright, something went wrong when rebasing and it looks like your commit is basically reverting some of the changes that were done to run_interop_tests.py recently (not sure how those changes got there). Please fix.